README.md

```diff
@@ -131,6 +131,6 @@ The license below refers only to the script and not to the downloaded content.
 
 ![broken](readme/status.gif)
 
-It's broken, I will fix it, maybe...
+~~It's broken, I will fix it, maybe...~~
```

Comment:
I would remove it totally from the README. No need to mention at all. What we could place here is that the script was fixed + date when we fixed it.